docs/project/2024/matei_teodor.petrec/index.md

```diff
@@ -0,0 +1,115 @@
+
+## Motivation
+
+I chose to work on the Tic-Tac-Toe project because it perfectly combines my love for classic games with my growing interest in technology and programming. The project, powered by the Raspberry Pi Pico, allows me to explore the complexity of electronic components and software design in a hands-on manner. I’m particularly excited about the challenge of integrating a joystick, wifi module of the pico W and a display, which adds a physical interaction element to the digital game. This test is not just about recreating a game I enjoy and I thought about because I like Ancient Egypt, it's about pushing my abilities in software and hardware design to new limits and crafting an enjoyable experience that enhances my rust skills and provides a tangible, fun product to bring people together.
+
+## Architecture 
+
+![Architecture photo](./architechture.jpg)
+
+The diagram shows the setup of the Interactive Tic-Tac-Toe Game System, built with the Raspberry Pi Pico W. This is a list with all the components and their functionality presented in the representation:
+
+1) Raspberry Pi Pico W: The central controller for the Tic-Tac-Toe game. | Manages input and output operations and processes game logic.
+It initializes the game by setting up the LCD display and configuring initial states for LEDs and the joystick/phone control.
+
+2) Joystick: Serves as the primary input device for the player. | Connected directly to the Raspberry Pi Pico W, allowing the user to navigate the game interface. | Has the functionality when pressed to confirm selected move on the screen.
+
+3) Phone (via CYW43439): Connects to the Raspberry Pi Pico W through the CYW43439 WiFi chip.
+The inclusion of the CYW43439 allows the game to have connectivity feature to transmit game input control from phone (←↑↓→)
+
+4) 240x240 IPS LCD: Displays the game state and user interface. | Connected to the Raspberry Pi Pico W via SPI (Serial Peripheral Interface), facilitating fast data transfer for real-time display updates. | 
+
+5) Buzzer: Provides auditory feedback for game events, such as invalid moves or game conclusions. | Triggered by specific game conditions programmed into the Raspberry Pi Pico W.
+
+6) LEDs: Indicate various game states and functions, such as player turns or game outcomes. | Controlled directly by the Raspberry Pi Pico W to reflect real-time game changes.
+
+
+<!-- ![3D Schematic](./obj.mtl)
+![3D Schematic in TinkerCad](./tinker.obj) -->
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+Raspberry Pi Pico W: Manages all game operations and processes inputs and outputs.
```

Comment:
Does this display correctly? Did you try it?

Reply:
I don't know, I didn't try it.